### PR TITLE
Added upgrade and uninstall guide to INSTALL.md

### PR DESCRIPTION
**Pull request by oleg68**
Resolves: #720

### CODE REVIEW

**Review by larspalo — CHANGES_REQUESTED** [review]
A few spelling mistakes should perhaps be fixed.

Line 4: general, upgrading
Line 89: already

**Review by Jean-Gabriel1982 — DISMISSED** [review]

File: INSTALL.md [hunk 1]
```diff
@@ -1,55 +1,74 @@
...
-In geneneral for installing GrandOrgue you have to do two steps:
+In geneneral for installing GrandOrgue or upgdaring from a previous version you have to do two steps:
 
-1. Download installation files from Github.
+1. Download an installation file from Github.
     * For a release version go to [https://github.com/GrandOrgue/grandorgue/releases](https://github.com/GrandOrgue/grandorgue/releases) and select a release number.
     * For an intermediate build go to [https://github.com/GrandOrgue/grandorgue/actions](https://github.com/GrandOrgue/grandorgue/actions), select a build and scroll down to `Artifacts`
     The necessary file names to download depend on the target operation system.
-2. Install the downloaded files. This step is operation system dependent.
+2. Install the downloaded files or upgrade existing installation with this file. This step is operation system dependent.
```
Comment (rousseldenis) [inline INSTALL.md:10]:
```suggestion
2. Install the downloaded file or upgrade existing installation with this file. This step is operation system dependent.
```

File: INSTALL.md [hunk 2]
```diff
@@ -1,55 +1,74 @@
...
 
 These two steps will be described later in the text.
 
-## Installation on Linux
-### Installation on rpm-based systems (centos, fedora, redhat, oracle linux, opensuse)
-1. Download the linux package ``grandorgue-<version>.<arch>.rpm`` where `<arch>` is
+## Attention
```
Comment (rousseldenis) [inline INSTALL.md:14]:
```suggestion
## Warning
```

File: INSTALL.md [hunk 3]
```diff
@@ -1,55 +1,74 @@
...
+
+Installation of program creates neither the configuration file nor directories used by GrandOrgue to store data and cache. These directories are reside in the user home directory and are created when GrandOrgue is launched the first time.
```
Comment (rousseldenis) [inline INSTALL.md:16]:
```suggestion
Installation of program creates neither the configuration file nor directories used by GrandOrgue to store data and cache. These directories reside in the user home directory and are created when GrandOrgue is launched the first time.
```

File: INSTALL.md [hunk 4]
```diff
@@ -58,30 +77,29 @@ These two steps will be described later in the text.
...
     ```
     apt-get install <path>/grandorgue_<version>_<arch>.deb
     ```
 
-4. After installation, run GrandOrgue from the Aplication menu or from the overview screen.
+4. After installation or upgrading, run GrandOrgue from the Aplication menu or from the overview screen.
 
 ### Running GrandOrgue on Linux without installation
-This method requires that all dependencies have already been installed. Usually this way is used for trying a new GrandOrgue version when an old one has been installed.
+This method requires that all dependencies have already been installed. Usually this way is used for trying a new GrandOrgue version when an old one has alreade been installed.
```
Comment (rousseldenis) [inline INSTALL.md:89]:
```suggestion
This method requires that all dependencies have already been installed. Usually this way is used for trying a new GrandOrgue version when an old one has already been installed.
```

File: INSTALL.md [hunk 5]
```diff
@@ -91,4 +109,45 @@ This method requires that all dependencies have already been installed. Usually
...
+    apt-get remove grandorgue
+    ```
+
+## Deinstallation on Windows
+
+1. Run ``Control Pannel`` -> `Programs` -> ``Uninstall a progam``.
+2. Select ``GrandOrgue``.
+3. Follow the prompts.
+
+## Attention
```
Comment (rousseldenis) [inline INSTALL.md:150]:
```suggestion
## Warning
```

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Some comments

**Comment by oleg68** [comment]
Thanks, @larspalo and @rousseldenis. I've made all suggesed changes.

**Review by larspalo — APPROVED** [review]

**Review by Jean-Gabriel1982 — APPROVED** [review]

**Comment by oleg68** [comment]
@rousseldenis could you check this PR?

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**